spec/fixpoint_spec.rb

```diff
@@ -14,80 +14,80 @@
 
   it 'raises if there is no fixpoints path' do # this test must come first, so we do not destroy fixpoints...
     FileUtils.rm_rf(fixpoints_path)
-    expect { described_class.from_database.save_to_file('fp_one') }.to raise_error(Fixpoint::Error)
+    expect { described_class.from_database(ActiveRecord::Base.connection).save_to_file('fp_one') }.to raise_error(Fixpoint::Error)
```

Comment:
Would it make sense to set a variable or method for this test to return `ActiveRecord::Base.connection` and just pass in `conn` or something similar to all fo these tests?